### ISSUE TITLE
[BUG] warnings in weakly_connected_components tests

### ISSUE THREAD

**Issue by afender** [rapidsai/cugraph#352]
```
/home/afender/sw/cugraph/cpp/src/tests/components/con_comp_test.cu(89): warning: variable "stream" was declared but never referenced

/home/afender/sw/cugraph/cpp/src/tests/components/con_comp_test.cu(53): warning: function "<unnamed>::Usecase::operator=" was declared but never referenced
```

**Comment by seunghwak** [comment]
When fixing this,

we should update line 98 to print the file name as well. Currently, it requires an additional step to figure out the missing file when file open fails.

```
 97     FILE* fpin = fopen(param.get_matrix_file().c_str(),"r");
 98     ASSERT_TRUE( fpin != nullptr );
=>      ASSERT_NE(fpin, nullptr) << "fopen (" << param.get_matrix_file().c_str() << ") failure.";
```

**Comment by afender** [comment]
closed by #444
